### PR TITLE
[WIP] Build problems

### PR DESCRIPTION
**Pull request by pared**
* [ ] Have you followed the guidelines in our
      [Contributing document](https://dvc.org/doc/user-guide/contributing)?

* [ ] Does your PR affect documented changes or does it add new functionality
      that should be documented? If yes, have you created a PR for
      [dvc.org](https://github.com/iterative/dvc.org) documenting it or at
      least opened an issue for it? If so, please add a link to it.

-----

### CODE REVIEW

**Comment by efiop** [comment]
looks like it might be a hidden import in one of the google packages. Try defining hidden-imports with that module in one of the google hooks that we have.

**Comment by efiop** [comment]
@pared Btw, you can see in the logs https://travis-ci.com/iterative/dvc/jobs/229336938 that your hook is not getting loaded, so I think the suspicion about hidden imports is right. Try adding hiddenimports with resumable media to https://github.com/iterative/dvc/blob/master/scripts/hooks/hook-google.cloud.storage.py (is it the one that has resiumable-media as a dependency?).

**Comment by pared** [comment]
adding hook for resumable media seems to help. Though, further we encounter UnknownExtra error for `requests`.

related: https://github.com/pyinstaller/pyinstaller/issues/3033

**Comment by pared** [comment]
Current solution is only workaround.
Need to pinpoint which `google-resumable-media` v`0.3.3` dependencies cause builds to fail.

**Comment by pared** [comment]
Seems to me that cause of our problems lies here:
- [google-resumable-media requiring requests](https://github.com/googleapis/google-resumable-media-python/blob/68603dbdfb3d6e9b325dcbe5acce29bc5455b3fe/google/resumable_media/requests/__init__.py#L665) in `0.3.2` 
- [after refactoring](https://github.com/googleapis/google-resumable-media-python/blob/c8373160c9fe43981ea42fd1d8e10abb8ddbe350/google/resumable_media/requests/__init__.py#L663) in `0.3.3`

So basically the problem is that `requests` become there `google.resumable_media.requests`. And PyInstaller is not aware of that.

**Comment by efiop** [comment]
@pared if that is so, then adding `hiddenimports = ['requests']` to google-resumable-media hook should help, no?

**Comment by pared** [comment]
@efiop 
Regretfully, it does not.

File: setup.py [hunk 1]
```diff
@@ -71,11 +71,7 @@ def run(self):
 
 # Extra dependencies for remote integrations
 
-# Restricting google-resumable-media version is due to the fact that
-# since 0.3.3 our binaries built by pyinstaller started to fail on Linux and
-# macOS. This restriction will be removed after
-# https://github.com/iterative/dvc/issues/2441 is resolved.
-gs = ["google-resumable-media==0.3.2", "google-cloud-storage==1.13.0"]
+gs = ["google-resumable-media>=0.4.0", "google-cloud-storage>=1.19.0"]
```
Comment (efiop) [inline setup.py]: I guess we no longer need to specify grm manually as long as we fix gcs.
```suggestion
gs = ["google-cloud-storage==1.19.0"]
```

Comment (efiop) [reply setup.py]: https://github.com/googleapis/google-cloud-python/blob/master/storage/setup.py#L34
```
    "google-resumable-media >= 0.4.0",
```
already specified in it's reqs.

File: setup.py [hunk 2]
```diff
@@ -107,7 +103,6 @@ def run(self):
     "mock>=3.0.0",
     "xmltodict>=0.11.0",
     "awscli>=1.16.125",
-    "google-compute-engine",
```
Comment (efiop) [inline setup.py]: Btw, I think I remember adding this one because of binary built by pyinstaller, when using it with gs remote. Can we try rather fixing the version of this package instead of removing it? Just to be safe.

Comment (pared) [reply setup.py]: hmm, is it possible to check which version will be the "good" one? Or latest should be enough?
